src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowController.cs

```diff
@@ -86,7 +86,7 @@ public async Task<IActionResult> Index(int workflowTypeId, WorkflowIndexViewMode
 
             var workflowType = await _workflowTypeStore.GetAsync(workflowTypeId);
 
-            var query = _session.Query<Workflow, WorkflowIndex>();
+            var query = _session.QueryIndex<WorkflowIndex>();
             query = query.Where(x => x.WorkflowTypeId == workflowType.WorkflowTypeId);
```

Comment:
These lines should possibly be merged into : 
`_session.QueryIndex<WorkflowIndex>(index => index.WorkflowTypeId == workflowType.workflowTypeId)`